internal/k8s/controllers/service_controller.go

```diff
@@ -67,6 +68,11 @@ func (r *ServiceReconciler) reconcileService(ctx context.Context, req ctrl.Reque
 		return ctrl.Result{}, err
 	}
 
+	if filterByLoadbalancerClass(service, r.LoadBalancerClass) {
+		level.Info(r.Logger).Log("controller", "ServiceReconciler", "filtered service", req.NamespacedName)
```

Comment:
wdyt about having this and the other Debug? I'm inclined to think someone using this will apply the class to a small subset of services and could easily have a lot of these logs around.